src/jdk.jconsole/share/classes/sun/tools/jconsole/resources/messages_de.properties

```diff
@@ -282,7 +282,7 @@ USERNAME_ACCESSIBLE_NAME=Benutzername
 USER_DATA=UserData
 VIRTUAL_MACHINE=Virtuelle Maschine
 VM_ARGUMENTS=VM-Argumente
-VMINTERNAL_FRAME_ACCESSIBLE_DESCRIPTION=Innerer Frame für die Überwachung einer Java Virtual Machine
+VMINTERNAL_FRAME_ACCESSIBLE_DESCRIPTION=Innerer Frame für das Monitoring einer Java Virtual Machine
```

Comment:
This one-off change is consistent with the change in _src/jdk.jdi/share/classes/com/sun/tools/example/debug/tty/TTYResources_de.java_.

Reply:
Consistently changed to be "[Denglisch](https://en.wikipedia.org/wiki/Denglisch)" instead of German.